### PR TITLE
Reliability depends part type

### PR DESCRIPTION
**Pull request by Gordon-Dry**
Reliability: mass and cost difference between standard and high quality is now relative to the part type

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
  * Added LH2+O2 processes to the Fuel Cells for USI and CryoTanks Support (PiezPiedPy)
  * Added Hydrogen Liquefaction and Liquid Hydrogen Evaporator processes to USI and CryoTanks Support (PiezPiedPy)
  * Fixed CryoTanks NRE in Planner and added LH2 to radial tanks for CryoTanks Support (PiezPiedPy)
+ * Reliability: mass and cost difference between standard and high quality is now relative to the part type (Gordon Dry)
```
Comment (PiezPiedPy) [inline CHANGELOG.md]: Most recent changes go at the top 😉 no need to change your PR though, just FYI

Comment (Gordon-Dry) [reply CHANGELOG.md]: thanks, I will have that in mind

**Comment by Gordon-Dry** [comment]
@steamp0rt Please wait until I think it's ready.
Still finding quirks.

**Comment by Gordon-Dry** [comment]
Looks much better now.
I will test a few dozen parts in VAB if everything is settled.

**Comment by Gordon-Dry** [comment]
Some sleep for me now

**Comment by Gordon-Dry** [comment]
Good to go, I tested a stack of 50 different parts plus radially attached parts in VAB, all look good now.

**Comment by PiezPiedPy** [comment]
me like

**Pull request merged**